interfaces/prompting/prompting.go

```diff
@@ -22,8 +22,25 @@ package prompting
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 	"strconv"
+	"sync/atomic"
 	"time"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/snapcore/snapd/logger"
+)
+
+const (
+	// maxIDFileSize should be enough bytes to encode the maximum prompt ID.
+	maxIDFileSize int = 8
+)
+
+var (
+	ErrExpirationInThePast = fmt.Errorf("cannot have expiration time in the past")
```

Comment:
this probably merits a comment that it could get wrapped?

Reply:
Yes, will add, thank you